src/MusicalScore/Graphical/VexFlow/VexFlowGraphicalNote.ts

```diff
@@ -76,19 +76,16 @@ export class VexFlowGraphicalNote extends GraphicalNote {
      * @param note
      * @param index
      */
-    public setIndex(note: Vex.Flow.StemmableNote, index: number): void {
+    public setIndex(note: StemmableNote, index: number): void {
         this.vfnote = [note, index];
         this.vfnoteIndex = index;
     }
 
-    public notehead(vfNote: Vex.Flow.StemmableNote = undefined): {line: number} {
-        let vfnote: any = vfNote;
-        if (!vfnote) {
-            vfnote = (this.vfnote[0] as any);
-        }
-        const noteheads: any = vfnote.note_heads;
+    public notehead(vfNote?: StemmableNote): {line: number} {
+        const vfnote: StemmableNote = vfNote || this.vfnote[0];
+        const noteheads: NoteHead[] = (vfnote as StaveNote).noteHeads;
         if (noteheads && noteheads.length > this.vfnoteIndex && noteheads[this.vfnoteIndex]) {
-            return vfnote.note_heads[this.vfnoteIndex];
+            return { line: (vfnote as StaveNote).noteHeads[this.vfnoteIndex].getLine() };
```

Comment:
@sschmidTU this is doing something different, makes sense?

Reply:
looks fine for now, as long as we only need the line from the Vexflow NoteHead, which is all we're using for now in VexFlowMeasure. I guess the old version doesn't work anymore because Vexflow now returns NoteHead instead of any :)

Reply:
ok